src/process/api/index.js

```diff
@@ -3,13 +3,20 @@ import axios from 'axios';
 const baseUrl = `${process.env.REACT_APP_API_URL}${process.env.REACT_APP_API_PATH_PREFIX}`;
 
 const api = axios.create({
-  baseURL: `${baseUrl}`
+  baseURL: `${baseUrl}`,
```

Comment:
Just paste in here the baseUrl, don't define it into a const

Reply:
gen asa?